oxidation/libparsec/crates/types/tests/test_certif.rs

```diff
@@ -475,3 +475,106 @@ fn serde_realm_role_certificate_no_role(alice: &Device, bob: &Device) {
     .unwrap();
     assert_eq!(certif2, expected);
 }
+
+#[rstest]
+fn serde_sequester_authority_certificate(alice: &Device) {
```

Comment:
Is there any script to generate these input data?

Reply:
Uhoh I don't know where it goes